source/tutorial/create-system-dsn.txt

```diff
@@ -31,6 +31,10 @@ data from MongoDB.
      - id: macos
        content: |
 
+     - id: linux
+       name: Linux
+       content: |
+
 Prerequisites
 -------------
 
```

Comment:
I think we should change the prerequisites to "Download and extract the MongoDB ODBC Driver..." for the linux tab.

Reply:
Done.